scripts/assets/test_file_exemptions.textproto

```diff
@@ -633,6 +633,9 @@ exempted_file_path: "utility/src/main/java/org/oppia/android/util/logging/fireba
 exempted_file_path: "utility/src/main/java/org/oppia/android/util/logging/firebase/FirebaseLogUploader.kt"
 exempted_file_path: "utility/src/main/java/org/oppia/android/util/logging/firebase/FirebaseLogUploaderModule.kt"
 exempted_file_path: "utility/src/main/java/org/oppia/android/util/logging/firebase/LogReportingModule.kt"
+exempted_file_path: "utility/src/main/java/org/oppia/android/util/networking/NetworkConnectionUtil.kt"
+exempted_file_path: "utility/src/main/java/org/oppia/android/util/networking/NetworkConnectionUtilDebugModule.kt"
+exempted_file_path: "utility/src/main/java/org/oppia/android/util/networking/NetworkConnectionUtilProdModule.kt"
```

Comment:
now we can remove these as we have test files

Reply:
Ah! Actually, we don't have tests for these files. These are interfaces and dagger modules only.

Reply:
I suggest adding tests for the modules. They can be simple tests which just verify that the correct implementation is injected, but modules are arranging a very specific configuration & it's useful to verify that configuration is correct.

Reply:
Tests added. PTAL.